packages/@webex/plugin-meetings/test/unit/spec/locus-info/index.js

```diff
@@ -2392,25 +2392,22 @@ describe('plugin-meetings', () => {
           };
         });
 
-        it('applyLocusDeltaData gets full locus on DESYNC action if we do not have a syncUrl and destroys the meeting if that fails', () => {
+        it('applyLocusDeltaData gets full locus on DESYNC action if we do not have a syncUrl and destroys the meeting if that fails', async () => {
           meeting.meetingRequest.getLocusDTO.rejects(new Error('fake error'));
 
           locusInfo.locusParser.workingCopy = {}; // no syncUrl
 
-          // Since we have a promise inside a function we want to test that's not returned,
-          // we will wait and stub it's last function to resolve this waiting promise.
-          return new Promise((resolve) => {
-            webex.meetings.destroy.callsFake(() => resolve());
-            locusInfo.applyLocusDeltaData(DESYNC, fakeLocus, meeting);
-          }).then(() => {
-            assert.calledOnceWithExactly(meeting.meetingRequest.getLocusDTO, {url: 'fullSyncUrl'});
+          locusInfo.applyLocusDeltaData(DESYNC, fakeLocus, meeting);
 
-            assert.notCalled(meeting.locusInfo.handleLocusDelta);
-            assert.notCalled(meeting.locusInfo.onFullLocus);
-            assert.notCalled(locusInfo.locusParser.resume);
```

Comment:
this test had a check for `resume()` not being called, but was passing despite the fact that `resume()` was called, because the test was missing a call to `testUtils.flushPromises()`, so now I've added it.